app/Models/BeatmapsetEvent.php

```diff
@@ -106,6 +106,17 @@ public static function search($rawParams = [])
             }
         }
 
+        if (isset($rawParams['beatmapset'])) {
```

Comment:
it should probably still check for empty string as in `present($rawParams['beatmapset'] ?? null)` to match behavior of other param